### ISSUE TITLE
CLI Incompatible with Ubuntu Server 22.04

### ISSUE THREAD

**Issue by soccerdude288** [bitwarden/clients#2726]
## Describe the Bug

**EDIT: 8/15/2022** 
Still a problem with Ubuntu: 22.04.01 and Bitwarden CLI: 2022.08.0

**EDIT: 4/24/2022** - We've realized that this issue only is a problem with secure notes inside an Organization Collection

Using the CLI on system running Ubuntu 22.04 to get notes returns error `[error: cannot decrypt]`

I've tried testing with the snap install, npm install, and native executable.

## Steps To Reproduce

Two docker images have been created to quickly show the problem. **Be sure to replace 123-123... with the actual id of the item.**

[Docker Image](https://github.com/soccerdude288/BitwardenCLIDockerImages/blob/main/Dockerfile-node) based on Node that DOES NOT have the problem.

`docker run -it --rm -v "$HOME/.config/Bitwarden CLI":"/root/.config/Bitwarden CLI" -e BW_SESSION caffeine01/bw get notes 123-123-123-123-123`

[Docker Image](https://github.com/soccerdude288/BitwardenCLIDockerImages/blob/main/Dockerfile-22.04) based on Ubuntu 22.04 that DOES have the problem

`docker run -it --rm -v "$HOME/.config/Bitwarden CLI":"/root/.config/Bitwarden CLI" -e BW_SESSION caffeine01/bw-ubuntu-22.04 get notes 123-123-123-123-123`

For those that don't want to use docker... I also tested with 2 barebone VM installs using 22.04 and 21.10

## Expected Result

Returns the value of the node

## Actual Result

Returns `[error: cannot decrypt]`

## Screenshots

Working...
![image](https://user-images.githubusercontent.com/17014310/164759377-bfbcb54d-4d8a-4245-a9de-46ac3b654e85.png)

Not working...

![image](https://user-images.githubusercontent.com/17014310/164761649-bffe754f-5cbf-444a-82ae-e1944e1200ce.png)

## Environment

- Operating system: Ubuntu LTS 22.04
- Shell: bash
- Build Version: 1.22.0

## Additional Context

It appears items like username and password work, but as far as I can tell, notes and name don't work. There may be more properties, but I haven't looked further.

My current workaround is to alias bw to the working docker image.
`alias bw='docker run -it --rm -v "$HOME/.config/Bitwarden CLI":"/root/.config/Bitwarden CLI" -e BW_SESSION caffeine01/bw:latest'`

[Also posted in Bitwarden Community](https://community.bitwarden.com/t/ubuntu-22-04-lts-bitwarden-cli-cant-decrypt-notes/40460)

**Comment by tangowithfoxtrot** [comment]
Thank you for the additional information, @Caffeine-01. I was able to reliably reproduce this issue with an Ubuntu 22.04 VM by trying to access _organization_ notes and item names. I have updated the status of this report in our internal bug-tracking system to "vetted". Thanks for reporting!

**Comment by soccerdude288** [comment]
@tangowithfoxtrot do you have any ETA on a fix for this? Although we have a workaround, it's not great. Thanks for your help.

**Comment by MAmlung** [comment]
Are there any news with that issue?

**Comment by soccerdude288** [comment]
Can confirm this is still a problem on Ubuntu: 22.04.01 and Bitwarden Cli: 2022.8.0

**Comment by dbosompem** [comment]
Hi all, I understand how frustrating this might be; sincere apologies! The team will try and expedite things so we can appropriately prioritise this and get it fixed.

**Comment by edupr91** [comment]
Any update on this issue? as this client also is being used from the SaaS version. this issue should be taken care of...

**Comment by skeetmtp** [comment]
Same here ! Hope the team will be able to work on the topic soon.

**Comment by soccerdude288** [comment]
@dbosompem any update on the timeline here?

**Comment by skeetmtp** [comment]
@soccerdude288 Are you still able to reproduce ?
Just tested again in a fresh VM with success!

versions:
bw version 2022.10.0 installed with npm install -g @bitwarden/cli
Ubuntu 22.04.1 LTS https://app.vagrantup.com/ubuntu/boxes/jammy64

**Comment by Caffeine-01** [comment]
I am still getting the error with notes on **Organization Collection**.  @skeetmtp  did you test with **Organization Notes**. Regular notes work.

```bash
$ bw get notes XXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
[error: cannot decrypt]

$ bw --version
2022.10.0

$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.1 LTS
Release:        22.04
Codename:       jammy
```

**Comment by nekhebit** [comment]
> Indeed I was not checking deep enough :( This command produced some lines:
> 
> ```
> bw list items | grep "cannot decrypt"
> ```

I bypass it like this:

`OPENSSL_CONF=/dev/null bw list collections | jq`

**Comment by Caffeine-01** [comment]
> `OPENSSL_CONF=/dev/null bw list collections | jq`

@nekhebit, I can confirm this workaround. I've now added `alias bw='OPENSSL_CONF=/dev/null bw'` to my .bash_aliases file.

Thank you.

**Comment by irvintim** [comment]
I am unable able get the "collection" names at all using the CLI:

`
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.1 LTS
Release:	22.04
Codename:	jammy

$ grep -B 2 'providers =' /etc/ssl/openssl.cnf

[openssl_init]
#providers = provider_sect

$ bw --version
1.22.1

$ bw list org-collections --organizationid XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX | jq '.[0]'
{
  "object": "collection",
  "id": "YYYYYYYY-YYYY-YYYY-YYYY-YYYYYYYYYYYY",
  "organizationId": "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX",
  "name": "[error: cannot decrypt]",
  "externalId": null
}

$ OPENSSL_CONF=/dev/null bw list org-collections --organizationid XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX | jq '.[0]'
{
  "object": "collection",
  "id": "YYYYYYYY-YYYY-YYYY-YYYY-YYYYYYYYYYYY",
  "organizationId": "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX",
  "name": "[error: cannot decrypt]",
  "externalId": null
}

$ bw get collection --organizationid XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX YYYYYYYY-YYYY-YYYY-YYYY-YYYYYYYYYYYY
{"object":"collection","id":"YYYYYYYY-YYYY-YYYY-YYYY-YYYYYYYYYYYY","organizationId":"XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX","name":"[error: cannot decrypt]","externalId":null}

$ OPENSSL_CONF=/dev/null bw get collection --organizationid XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX YYYYYYYY-YYYY-YYYY-YYYY-YYYYYYYYYYYY
{"object":"collection","id":"YYYYYYYY-YYYY-YYYY-YYYY-YYYYYYYYYYYY","organizationId":"XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX","name":"[error: cannot decrypt]","externalId":null}

$ bw get org-collection --organizationid XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX YYYYYYYY-YYYY-YYYY-YYYY-YYYYYYYYYYYY
encValue.split is not a function

$ OPENSSL_CONF=/dev/null bw get org-collection --organizationid XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX YYYYYYYY-YYYY-YYYY-YYYY-YYYYYYYYYYYY
encValue.split is not a function

`

**Comment by csomh** [comment]
This seems to be fixed now with version 2023.1.0, at least it works now on Fedora Linux 37.

**Comment by Krychaz** [comment]
Hello everyone. 

The issue should be fixed by now. If anyone is still experiencing issues related to this, please let us know.
